data/scripts.js

```diff
@@ -1174,6 +1174,23 @@ exports.BattleScripts = {
 		}
 
 		let movePool = (template.randomBattleMoves ? template.randomBattleMoves.slice() : Object.keys(template.learnset));
+
+		// Check for alternate move pool.
```

Comment:
This is perhaps the wrong position for it. This also runs a useless `Object.keys(template.learnset)` before replacing `movePool`, and would be best off being rewritten not to need that.